Documentation/CONFIGURATION.md

```diff
@@ -0,0 +1,161 @@
+# Configuring the Fastly_Cdn Module
+
+This document describes how to configure the Fastly_Cdn module and connect it
+to Fastly's CDN services.
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Configure the Fastly Service](#configure-the-fastly-service)
+* [Basic Configuration](#configure-the-module)
+* [Further Configuration Options](#further-configuration-options)
+
+## Prerequisites
+
+To proceed with configuration you will need to have
+[signed up for Fastly's service](https://www.fastly.com/signup). You will need
+the API key (or a user key with purge access). Details of how to find these
+are kept in [Fastly's documentation](https://docs.fastly.com/guides/account-management-and-security/finding-and-managing-your-account-info).
+
+## Configure the Fastly Service
+
+To start configuring the Fastly service for use with Magento ensure that you
+have [signed up](https://www.fastly.com/signup) for the service. Once you have
+confirmed the account and logged in the Fastly application will take you to a
+wizard to configure your first service. This is further documented on [Fastly's documentation](https://docs.fastly.com/guides/basic-setup/sign-up-and-create-your-first-service).
+
+You will need to email Fastly's support team to
+[request access](https://docs.fastly.com/guides/vcl/uploading-custom-vcl) to
+upload VCL to your account.
+Once this is enabled refresh the Fastly application and navigate to the VCL
+page in the latest unlocked version of the service. Once there upload the
+[Fastly_Cdn VCL](../etc/fastl.vcl) as the main VCL. Review the generated VCL
+to ensure that it is correctly implemented. Then activate the new version.
+
+## Configure the Module
+
+This section will describe how to input the API key and service ID into the
+Fastly_Cdn module. It assumes that the module has already been installed
+without error. If not please see [installing the module](INSTALLATION.md) for
+details of how to do this.
+
+To enter the credentials for the Fastly_Cdn module to use open a browser and
+log in to the admin section of the Magento server. Navigate to:
+
+```
+Stores > Configuration > Advanced > System
+```
+
+Expand the 'Full Page Cache' section then choose 'Fastly CDN' in the 'Caching
+Application' dropdown.
+
+Adjust the TTL for public content to suit the needs of the server. (This value
+is in seconds.)
+
+Expand the 'Fastly Configuration' section to enter the remaining necessary
+details.
+
+Leave the 'Fastly API Endpoint' set as the default value.
```

Comment:
Required to pass Magento module certification, so need to document not to change it.